brayns/engine/framebuffer/types/StaticFrameHandler.cpp

```diff
@@ -23,35 +23,96 @@
 #include <brayns/engine/framebuffer/ToneMapping.h>
 #include <ospray/ospray_cpp/Data.h>
 
+#include <algorithm>
+
 namespace
 {
+struct FrameBufferParameters
+{
+    inline static const std::string operations = "imageOperation";
+};
+
 /**
  * @brief Returns the size, in bytes, of each color channel of a given pixel format
  */
 class PixelFormatChannelByteSize
 {
 public:
-    static size_t get(brayns::PixelFormat format)
+    static size_t get(brayns::FramebufferChannel channel, brayns::PixelFormat format)
     {
-        switch (format)
+        if (channel == brayns::FramebufferChannel::Color)
```

Comment:
Just maniac stuff but I would put if channel != Color return 4 and then the rest...

Reply:
Addressed